reference-architecture/day2ops/scripts/ocp36-sat6.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# vim: sw=4 ts=4 et
+import os, argparse, socket, getpass, subprocess
+
+class ocpSat6(object):
+
+    __name__ = 'ocpSat6'
+
+    openshift3Images=[]
+
+    def __init__(self, load=True):
+
+        if load:
+            self._parseCli()
+            self._loadImageList()
+            self._addData()
+            self._syncData()
+
+    def _loadImageList(self):
+        cmd='curl -s https://registry.access.redhat.com/v1/search?q="openshift3" | python -mjson.tool | grep ".name.:" | cut -d: -f2 | sed -e "s/ "//g"" -e "s/,"//g""'
+        result = subprocess.check_output(cmd, shell=True)
+        lines = result.splitlines()
+        for line in lines:
+            nl = line.replace('"', '')
+            self.openshift3Images.append(nl)
+
+    def _parseCli(self):
+
+        parser = argparse.ArgumentParser(description='Add all OCP images for disconnected installation to satellite 6', add_help=True)
+        parser.add_argument('--orgid', action='store', default='1',help='Satellite organization ID to create new product for OCP images in')
+        parser.add_argument('--productname', action='store', default='ocp36',help='Satellite product name to use to create OCP images')
+        parser.add_argument('--password', action='store', help='Password for hammer CLI')
+        self.args = parser.parse_args()
+
```

Comment:
I'd say to add --satellite to specify the satellite URL and --username for the user name, and if they are not specified, use the hammer defaults.
I'd also say it requires hammer to be installed in the host running the script somehow in the help

Reply:
The intent was to run it locally on a satellite server. But, I can add it as an argument and default it to localhost. The username if not specified defaults to admin. But, I can change that to the same as the server.

Reply:
```
[root@sat6 ~]# hammer --server localhost --password 100Root- organization list
Error: undefined method `request' for #<String:0x0000000231f1f8>
```
I'm going to add username and wait on --server